toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -169,6 +172,14 @@ func createImageCustomizerParameters(buildDir string,
 		return nil, fmt.Errorf("the output PXE artifacts directory ('--output-pxe-artifacts-dir') can be specified only if the output format is an iso image.")
 	}
 
+	// output artifacts
+	ic.outputArtifactsDir = config.Output.Artifacts.Path
```

Comment:
Why have the `outputArtifactsDir` field if it is just a copy of `config.Output.Artifacts.Path`?